scrapers/GirlsRimming.yml

```diff
@@ -0,0 +1,28 @@
+name: "GirlsRimming"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - girlsrimming.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class='updatesBlock']/h2[@class='title']/text()
+      Details: //meta[@name='description']/@content
+      Image: //div[@class='player-thumb']/img/@src0_1x
+      Studio:
+        Name:
+          fixed: GirlsRimming
+      Movies:
+        Name:
+          selector: //div[@class='updatesBlock']/h2[@class='title']/text()
+          postProcess:
+            - replace:
+              - regex: (.+)(?:\sEp\d:\s).+
+                with: $1
+      Tags:
+        Name:
+          selector: //meta[@name='keywords']/@content
```

Comment:
```yaml
          postProcess:
            - replace:
                - regex: "[^,]*Id\\s(\\d+)[^,]*"
                  with:
          split: ","
```
filters out from the tags some performer names that have ids (eg in `https://www.girlsrimming.com/tour/trailers/Fidelity-Ep1.html`)